src/toil/provisioners/aws/awsProvisioner.py

```diff
@@ -145,6 +145,20 @@ def _sshInstance(cls, leaderIP, *args):
         return ouput
 
     @classmethod
+    def rsyncLeader(cls, clusterName, args):
+        leader = cls._getLeader(clusterName)
+        logger.info("Rsync ready")
+
+        ssh_command = 'ssh -o "StrictHostKeyChecking=no"'  # Skip host key checking
+        remote_rsync = "docker exec -i leader rsync"  # Access rsync inside appliance
+        args = [i.replace(":", "core@%s:" % leader.ip_address) for i in args]  # Insert remote host address
```

Comment:
Ahh, but this is too aggressive. You also conveniently excluded colon from test set. Try to find a more surgical strategy for substituting the colon. You should only replace exactly once (it would indicate a user error to not replace at all), only in a positional argument and only at the argument's beginning.

See also http://unix.stackexchange.com/questions/11581/rsync-colon-in-file-names